planning/planning_scene_monitor/src/planning_scene_monitor.cpp

```diff
@@ -501,6 +502,8 @@ bool planning_scene_monitor::PlanningSceneMonitor::newPlanningSceneMessage(const
 
     last_update_time_ = ros::Time::now();
     last_robot_motion_time_ = scene.robot_state.joint_state.header.stamp;
+    ROS_DEBUG_STREAM_NAMED("PSM", "scene update " << fmod(last_update_time_.toSec(), 10.)
```

Comment:
Please rename all the stream output to "planning_scene_monitor". This named-log is already used in the psm and I don't see a reason to add two different names for one class.